jobs/build/rhcos/Jenkinsfile

```diff
@@ -87,7 +91,10 @@ node {
             if (multi_builds_enabled) {
                 currentBuild.displayName += "multi"
                 echo "triggering multi builds"
-                def jenkins_url = 'https://jenkins-rhcos.apps.ocp-virt.prod.psi.redhat.com'
+                def jenkins_url = 'https://jenkins-rhcos--prod-pipeline.apps.int.prod-stable-spoke1-dc-iad2.itup.redhat.com'
```

Comment:
Does this jenkins_url used?

Reply:
yes, it is used.

Reply:
I don't see this jenkins_url used in the Jenkinsfile